### PR TITLE
feat: [iOS] prevent stopping background music

### PR DESCRIPTION
**Pull request by dw2kim**
Set AvAudioSession to be ambient and mixWithOthers

### CODE REVIEW

File: ios/Classes/SwiftOcarinaPlugin.swift [hunk 1]
```diff
@@ -25,6 +25,13 @@ class LoopPlayer: Player {
         playerLooper = AVPlayerLooper(player: player, templateItem: playerItem)
         
         player.volume = Float(volume)
+        
+        do {
+            try AVAudioSession.sharedInstance().setCategory(AVAudioSession.Category.ambient, options: AVAudioSession.CategoryOptions.mixWithOthers)
+            try AVAudioSession.sharedInstance().setActive(true)
+        } catch let error {
+            print(error)
```
Comment (erickzanardo) [inline ios/Classes/SwiftOcarinaPlugin.swift:33]: Can't we have a better handling here? Maybe throw a platform exception, this way the error will fail silently.

**Review by erickzanardo — COMMENTED** [review]
This changes the behaviour of the plugin, I think this should be a parameter, with a default value of false.

Also, is there an equivalent of this on Android. I would like to avoid having features on one platform and not on the other.

**Comment by dw2kim** [comment]
That's correct. I was not thinking about Android atm. I will see what I can do for the Android.

Also, I will add the parameter and set it as false as a default value.

File: ios/Classes/SwiftOcarinaPlugin.swift (hunk 1 above)
Comment (dw2kim) [reply ios/Classes/SwiftOcarinaPlugin.swift:33]: Sure. Sounds good

**Comment by dw2kim** [comment]
Close this PR for now

**Pull request closed without merging**